frigate-beta/README.md

```diff
@@ -1,33 +1,36 @@
 # Home Assistant Add-on: frigate
 
-An Home Assitant Add-on for Frigate - Realtime Object Detection for IP Cameras  
+An Home Assistant Add-on for Frigate - Realtime Object Detection for IP Cameras
 All the heavy lifting is done by https://github.com/blakeblackshear/frigate
-Uses OpenCV and Tensorflow to perform realtime object detection locally for IP cameras. Designed for integration with HomeAssistant or others via MQTT.
+Uses OpenCV and Tensorflow to perform realtime object detection locally for IP
+cameras. Designed for integration with HomeAssistant or others via MQTT.
 
-![Supports aarch64 Architecture][aarch64-shield] ![Supports amd64 Architecture][amd64-shield] ![Supports armhf Architecture][armhf-shield] ![Supports armv7 Architecture][armv7-shield] ![Supports i386 Architecture][i386-shield]
+![Supports aarch64 Architecture][aarch64-shield]
+![Supports amd64 Architecture][amd64-shield]
+![Supports armhf Architecture][armhf-shield]
+![Supports armv7 Architecture][armv7-shield]
+![Supports i386 Architecture][i386-shield]
 
-This repository contains two add-ons:  
-frigate: uses the current production release 0.5.0  
-frigate-beta: uses the current beta release 0.4.0-beta
+This repository contains two add-ons:
+frigate: uses the current production release 0.5.0
+frigate-beta: uses the current beta release 0.5.4-rc4
```

Comment:
I think I would like to follow the release channels of the original project, so prod beta and rc

Reply:
Would be nice to figure out if we could just specify configuration setting for a docker tag and then just have one.